<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Toolbar</class>
 <widget class="QWidget" name="Toolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>50</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>50</width>
     <height>400</height>
    </rect>
   </property>
   <widget class="QPushButton" name="visibleAreaButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/visiblearea.png</normaloff>:/toolbar/visiblearea.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>21</width>
      <height>21</height>
     </size>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="lineButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>202</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/line.png</normaloff>:/toolbar/line.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="rectButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>152</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/rectangle.png</normaloff>:/toolbar/rectangle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="ellipseButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>102</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/circle.png</normaloff>:/toolbar/circle.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="textButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>302</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/text.png</normaloff>:/toolbar/text.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>14</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="okButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>352</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/ok.png</normaloff>:/toolbar/ok.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>29</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="arrowButton">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>252</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../resources/res.qrc">
      <normaloff>:/toolbar/arrow.png</normaloff>:/toolbar/arrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="selectedCircle">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>102</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dragLabel">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>52</y>
      <width>46</width>
      <height>46</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../resources/res.qrc">:/toolbar/drag.png</pixmap>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
